src/main/groovy/nebula/plugin/dependencylock/tasks/GenerateLockTask.groovy

```diff
@@ -105,6 +108,66 @@ class GenerateLockTask extends AbstractLockTask {
         }
     }
 
+    static Collection<Configuration> filterNonLockableConfigurationsAndProvideWarningsForGlobalLockSubproject(Project subproject, Set<String> configurationNames, Collection<Configuration> lockableConfigurations) {
```

Comment:
This method is used only in global locking and only when a user has declared specific configurations to lock. In this case, we provide warnings when those configurations are not going to be locked, so that users can remove or update `dependencyLock.configurationNames`